Source/Tenancy/IPerformAsynchronousActionOnAllTenants.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Threading.Tasks;
+using Dolittle.Execution;
+using Dolittle.Tenancy;
+
+namespace Dolittle.Runtime.Tenancy
+{
+    /// <summary>
+    /// Defines a system that performs an asynchronous action for <see cref="ITenants.All" /> in an <see cref="ExecutionContext" /> with the tenant set for each Tenant.
+    /// </summary>
+    public interface IPerformAsynchronousActionOnAllTenants
```

Comment:
Suggestion; would it be an idea to actually just merge this with the other interface - just keep one and then differentiate on having `Perform` and `PerformAsync`.  The `PerformInParallel` would just keep its name.

Reply:
Yeah could do that

Reply:
Fixed by https://github.com/dolittle-runtime/Runtime/pull/381/commits/efba7f4fccbaf107e0cd31ac1eba6948f06e5c0b